src/download.cpp

```diff
@@ -206,6 +206,13 @@ void DownloadFile(const std::string& url, IDownloadSink *sink, int flags)
 
     if ( !filename_set )
     {
+        char ioptionurl[512];
+        DWORD ousize = 512;
```

Comment:
`ousize` from `optionurl`. Not great, I know.

I noticed I also omitted the leading and trailing space in the 'if' statement